client/signup/steps/domains/index.jsx

```diff
@@ -246,9 +248,22 @@ class DomainsStep extends React.Component {
 		return this.props.designType;
 	};
 
+	shouldIncludeDotBlogSubdomain() {
+		const { flowName, siteGoals } = this.props;
+		const siteGoalsArray = siteGoals ? siteGoals.split( ',' ) : [];
+
+		return (
+			// 'subdomain' flow coming from .blog landing pages
+			flowName === 'subdomain' ||
+			// User picked only 'share' on the `about` step
+			( abtest( 'includeDotBlogSubdomain' ) === 'yes' &&
+				siteGoalsArray.length === 1 &&
```

Comment:
Wouldn't it be okay to show .blog subdomains even if they've selected other site goals in addition to selecting the `sharing` goal?

Reply:
In that case it's kind-of less a blog.

Reply:
This is the scope that @p3ob7o defined (see `p1530782141000346-slack-team-cobalt`), so let's stick with that for now 👍